lib/Zonemaster/Backend/DB/SQLite.pm

```diff
@@ -128,7 +128,7 @@ sub recent_test_hash_id {
 
     my $dbh = $self->dbh;
     my ( $recent_hash_id ) = $dbh->selectrow_array(
-        "SELECT hash_id FROM test_results WHERE fingerprint = ? AND test_start_time > DATETIME('now', ?)",
+        "SELECT hash_id FROM test_results WHERE fingerprint = ? AND creation_time > DATETIME('now', ?)",
```

Comment:
It is consistent with the other database modules to use "creation_time", but it is not logical. It is not the time of creation that makes sense, but the time of test execution. Are you sure that you do not want to change the other databases to use "test_start_time" instead?

Reply:
This is exactly the purpose of #840 to discuss which one should be used. However when working on #841, I chose to make SQLite consistent with MySQL and PostgreSQL so that the behavior is the same with all databases.

> Is this inconsistency bad enough to be included in the patch release?

Since this should have been part of v2021.2 release, I think this should be included in v2021.2.1.

Reply:
If we can be safe that there are no bad side effects then it could be included even though it probably has very low impact since very few batches are run.

Reply:
Since we reuse the same logic as the one with PostgreSQL and MySQL which have been around since the beginning, see commit ab540299f23 and 8b57fe576, I don't think this would have any bad side effects.